vortex-ffi/src/file.rs

```diff
@@ -152,7 +152,9 @@ pub unsafe extern "C" fn File_scan(
         stream = stream.with_projection(select(field_names, Identity::new_expr()));
     }
 
-    let stream = stream.build().vortex_expect("into_array_stream");
+    let stream = stream
+        .spawn_tokio(RUNTIME.handle().clone())
```

Comment:
@a10y let's discuss this and what we intend to happen. Really this is like using tokio as a thread pool. Speaking of... maybe we should add a spawn_rayon